.travis.yml

```diff
@@ -33,7 +36,7 @@ before_install:
   - ./travis/download_and_run_es.sh
 
 install:
-  - composer install --prefer-source
+  - composer install --prefer-dist
```

Comment:
source make no sense in ci and will end up in errors like the previous commits: https://travis-ci.org/elastic/elasticsearch-php/jobs/467216401